src/RA_Dlg_GameLibrary.cpp

```diff
@@ -47,8 +48,8 @@ bool ListFiles(std::string path, std::string mask, std::deque<std::string>& rFil
         directories.pop();
 
         WIN32_FIND_DATA ffd;
-        HANDLE hFind = FindFirstFile(NativeStr(spec).c_str(), &ffd);
-        if (hFind == INVALID_HANDLE_VALUE)
+        ra::FindFileH hFind{ ::FindFirstFile(NativeStr(spec).c_str(), &ffd) };
+        if (hFind.get() == INVALID_HANDLE_VALUE)
```

Comment:
As a test, I put the `.nes` and `.pas` (headerless ROM) extensions for RANes and it went pass this condition. I put it back and as expected the handle was invalid.